share/doc/wake/quickref.md

```diff
@@ -10,16 +10,28 @@
 the wake library (share/wake/lib) and all files *.wake in
 the wake.db directory and below.
 
-1. wake is invoked normally, although arguments are generally quoted.
+1. wake is invoked normally with a command-line target and its arguments.
+   Targets are any wake function in scope with a single `args: List String` parameter.
+    ```wake
+    wake runThing "a string"
     ```
-    wake 'runThing "a string"'
+
+1. Arbitrary wake expressions can be invoked with `-x`,
+   although the entire expression is generally quoted with single quotes to avoid interactions with the shell.
+    ```wake
+    wake -x 'runThing "a string"'
     ```
 
 1. Other useful flags
-    1. `wake -v <varible>` to get type information
-    1. `wake -o <file>` to get dependency information for job which write file
-    1. `wake -i <file>` to get dependency information for job which read file
-    1. `wake -g` To get a list of all global symbols
+    1. `wake -xv <varible>` to get type information for a variable or function that is in scope
+    1. Getting information about jobs. For all these commands, adding `-v` will
+       include the jobs' stdout/stderr, and adding `-d` will include the Wake
+       stack trace of when the job was run.
+        1. `wake -o <file>` to get dependency information for job which write file
```

Comment:
nit: can you back tick the variables, to make it more clear when they are being referenced?

Ex: `wake -o <file>` to get dependency information for job which write `file`

Reply:
I'm glad you asked, since I was also thinking of doing that, but wasn't sure how much unrelated stuff I should change in this PR. Updated in 39a63a48.